<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="colourPaletteGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Colour Palette</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="algorithmLabel">
        <property name="text">
         <string>Algorithm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="algorithmComboBox">
        <item>
         <property name="text">
          <string>Jenks</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Otsu</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="numMaterialsSpinBox">
        <property name="minimum">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="numMaterialsLabel">
        <property name="text">
         <string>Number of Materials</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="colourMapLabel">
        <property name="text">
         <string>Colour Map</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="colourMapComboBox">
        <item>
         <property name="text">
          <string>thermal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>flame</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>yellowy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>bipolar</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>spectrum</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cyclic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>greyclip</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>grey</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>viridis</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>inferno</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>plasma</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>magma</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
